app/models/help_us_improve_user.rb

```diff
@@ -0,0 +1,5 @@
+class HelpUsImproveUser
```

Comment:
Is this model actually used? It seems like we don't process this input on the backend, we just track this in GA... is that right?

Reply:
That's correct - I've now removed the unused model.